ovve_ui/ovve_ui.py

```diff
@@ -176,6 +178,15 @@ def setupLogging(self, logger, patient_id):
         if (self.dev_mode):
             logger.addHandler(ch)
 
+        self.main_timer = QTimer()
+        self.main_timer.start(60 * 1000)
+        self.main_timer.timeout.connect(self.addMinute)
+
+    def addMinute(self):
+        self.datetime = self.datetime.addSecs(60)
```

Comment:
Can you use the system time instead?  It might be incorrect vs real time, but you could store a constant offset when you set the clock initially.   Updating time this way seems like it will surely accumulate error over time.